source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,53 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+Pass the indexed ``name`` field to the ``dropIndex()`` method to remove
+an index from a collection. 
+
+Use the ``listIndexes()`` method to see the indexed ``name`` fields in
+your collection. 
+
+The following snippet retrieves all the indexes in your collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin listIndex
+   :end-before: end listIndex
+
+Your output should look something like the following:
+
+.. code-block:: json
+
+   { "v": 2, "key": {"_id": 1}, "name": "_id_" }
+   { "v": 2, "key": {"_fts": "text", "_ftsx": 1}, "name": "title_text", "weights": {"title": 1}, 
+   "default_language": "english", "language_override": "language", "textIndexVersion": 3 }
+
+The output preceding tells us there is a text index with the name "title_text".
```

Comment:
A couple nitpicks:
1) The ordering of the words "output preceding" requires an object whereas "preceding output" is an object. If you want to use "output preceding" you would follow with something like this: "The output preceding this sentence" which is a bit awkward.
2) I think the output isn't telling us something. Rather, "you can see a text index... in the preceding output".